packages/utils/package.json

```diff
@@ -10,31 +10,49 @@
     "dist/"
   ],
   "scripts": {
-    "test": "jest && yarn posttest",
+    "test:prepare": "yarn mkdirp test/fixtures && ./scripts/generate-fixtures.sh",
+    "test": "yarn test:prepare && jest && yarn posttest",
     "posttest": "jest-it-up",
     "test:ci": "yarn test",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' --ignore-path ../../.gitignore",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:changelog": "yarn auto-changelog validate",
+    "ajv-compile": "ajv compile --strict=true --all-errors",
+    "build:ajv": "./scripts/build-ajv.sh",
+    "build:pre-tsc": "yarn build:ajv && node scripts/transpileSchemaTypes.js",
     "build:tsc": "tsc --project tsconfig.local.json",
-    "build": "yarn build:tsc",
+    "build": "yarn build:pre-tsc && yarn build:tsc",
     "build:clean": "yarn clean && yarn build",
     "clean": "rimraf '*.tsbuildinfo' 'dist/*'",
     "publish:package": "../../scripts/publish-package.sh"
   },
   "dependencies": {
-    "@chainsafe/strip-comments": "^1.0.7"
+    "@chainsafe/strip-comments": "^1.0.7",
+    "@metamask/utils": "^2.0.0",
+    "ajv": "^8.11.0",
+    "concat-stream": "^2.0.0",
+    "fast-deep-equal": "^3.1.3",
```

Comment:
I removed some stuff that wasn't needed anymore, but I have no clue if some of those are used elsewhere or not.

Reply:
I think we should make sure the dependencies that aren't required anymore are removed, otherwise they'll stick around and no one will ever dare remove them 😅